src/cargo/core/compiler/context/mod.rs

```diff
@@ -72,18 +72,6 @@ impl<'a> Unit<'a> {
 	}
 }
 
-impl<'a> Ord for Unit<'a> {
```

Comment:
Out of curiosity, was this done for performance reasons or for correctness reasons?

Reply:
Yes, it is only for performance. The old code was rehashing every field for every comparison.